vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/DeviceSteps.java

```diff
@@ -118,6 +136,23 @@ public void pressKeys(ExamplesTable keys)
         pressKeys(keys.getColumn("key"));
     }
 
+    /**
+     * Types the text into the focused field
+     * @param text the text to type
+     */
+    @When("I type text `$text`")
```

Comment:
https://github.com/vividus-framework/vividus/blob/master/vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/TouchSteps.java#L104 -  I guess we need to have similar logic in both steps